### ISSUE TITLE
BYOR site not building a radar for  certain sheets

### ISSUE THREAD

**Issue by mjhaller** [thoughtworks/build-your-own-radar#132]
I can't seem to generate a radar for this sheet (or any variation of it), despite having some success a few days ago.

This is the sheet that does not work:
https://docs.google.com/spreadsheets/d/1u4tnw0x1TREDMSxW3cF1vG1WhMkhvUc5F-p29elbjrY/edit?usp=sharing

The example sheet will generate a radar.

**Comment by SARAN-thala** [comment]
Hi @mjhaller , 

I have copied this content and created a new google sheet. That sheet will render the radar perfectly. 
Could you explain what kind of changes did you make in that sheet, can help to debug this issue more?

**Comment by mjhaller** [comment]
Not sure I did anything.  I've tried creating new sheets with the same data in different browsers etc.  Still doesn't work.  Here is even a csv version that doesn't work.  https://radar.thoughtworks.com/?sheetId=https%3A%2F%2Fgitlab.com%2Fthezebra-public%2Ftech-radar%2Fraw%2Fmaster%2Ftech-radar.csv  

Although I believe it is CORS:
<img width="840" alt="image" src="https://user-images.githubusercontent.com/1828935/71988858-40f90980-31f6-11ea-9278-42815f06bcd4.png">

I don't know how to influence the CORS headers that google drive would give (or gitlab), so either we'll host it elsewhere, or I think I'll just run it locally.   

*Note: If we make a private sheet and authorize it does work, but we dont' want to authorize the tool to read all our documents in drive.*
